### PR TITLE
Removed lrelease and .qm files check-in from release process.

### PR DESCRIPTION
**Pull request by softins**
**Short description of changes**

Removed lrelease and .qm files check-in from release process. These steps no longer needed as .qm files are now compiled on the fly.

**Context: Fixes an issue? Related issues**

This update is needed due to the PR https://github.com/jamulussoftware/jamulus/pull/3288

**Status of this Pull Request**

Ready

**What is missing until this pull request can be merged?**

Nothing

**Does this need translation?**
NO

## Checklist

- [x] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/main/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
- [x] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. 
- [x] I'm sure that this Pull Request goes to the correct branch

### CODE REVIEW

**Review by pljones — APPROVED** [review]

**Comment by ignotus666** [comment]
@softins @pljones this stuff doesn't get translated.

File: contribute/en/Release-Process.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ Use `tools/create-translation-issues.sh` to create and assign issues (see usage
 {:.no_toc}
 
 Each translator should submit a PR containing just their new version of the `.ts` file. There is no need to submit
-a new `.qm` file, as they will all be regenerated below.
+a new `.qm` file, as they are now compiled automatically.
```
Comment (ann0see) [inline contribute/en/Release-Process.md]:
```suggestion
a new `.qm` file, as they are compiled automatically.
```

**Comment by softins** [comment]
> @softins @pljones this stuff doesn't get translated.

Ok, that's cool, I wasn't sure, having not made website changes before.

**Comment by pljones** [comment]
> @softins @pljones this stuff doesn't get translated.

The url contains `contribute/en` -- should it be outside the language-specified tree?

**Pull request merged**